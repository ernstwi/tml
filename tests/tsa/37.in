init foo
bar

foo -> bar
    guard { x < 5 }
    action { a }
    reset { x, y }

foo -> bar
    guard { x < 5 }
    action { a }
    reset { x, y }
