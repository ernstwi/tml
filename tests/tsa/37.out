{
    "actions": [
        "a"
    ],
    "clocks": [
        "x",
        "y"
    ],
    "edges": [
        {
            "action": "a",
            "from": "foo",
            "guard": "x<5",
            "reset": [
                "x",
                "y"
            ],
            "to": "bar"
        }
    ],
    "locations": [
        {
            "id": "bar",
            "initial": false,
            "invariant": null
        },
        {
            "id": "foo",
            "initial": true,
            "invariant": null
        }
    ]
}
